app/src/main/java/org/mozilla/fenix/settings/TrackingProtectionFragment.kt

```diff
@@ -6,47 +6,113 @@ package org.mozilla.fenix.settings
+        activity?.title = getString(R.string.preference_enhanced_tracking_protection)
         (activity as AppCompatActivity).supportActionBar?.show()
 
         // Tracking Protection Switch
         val trackingProtectionKey =
             context!!.getPreferenceKey(R.string.pref_key_tracking_protection)
         val preferenceTP = findPreference<SwitchPreference>(trackingProtectionKey)
 
-        preferenceTP?.isChecked = Settings.getInstance(context!!).shouldUseTrackingProtection
+        bindStrict()
+        bindRecommended()
+        setupRadioGroups()
+
+        val globalSetting = Settings.getInstance(context!!).shouldUseTrackingProtectionGlobal
+        preferenceTP?.isChecked = globalSetting
+
         preferenceTP?.onPreferenceChangeListener =
             Preference.OnPreferenceChangeListener { _, newValue ->
                 Settings.getInstance(requireContext())
-                    .setTrackingProtection(newValue = newValue as Boolean)
+                    .setTrackingProtectionGlobal(newValue = newValue as Boolean)
                 with(requireComponents) {
                     val policy = core.createTrackingProtectionPolicy(newValue)
                     useCases.settingsUseCases.updateTrackingProtection.invoke(policy)
                     useCases.sessionUseCases.reload.invoke()
                 }
+                radioStrict.isEnabled = newValue
+                radioStandard.isEnabled = newValue
                 true
             }
 
         val exceptions =
             context!!.getPreferenceKey(R.string.pref_key_tracking_protection_exceptions)
         val preferenceExceptions = findPreference<Preference>(exceptions)
         preferenceExceptions?.onPreferenceClickListener = getClickListenerForExceptions()
+        lifecycleScope.launch {
+            // TODO On first layout these views aren't inited yet for some reason
+            //  so I've added a delay before enabling/disabling them based on global setting
+            delay(ENABLED_DELAY)
+            radioStandard.isEnabled = globalSetting
+            radioStrict.isEnabled = globalSetting
+        }
+    }
+
+    private fun bindStrict() {
+        val keyStrict = getString(R.string.pref_key_tracking_protection_strict)
+        radioStrict = requireNotNull(findPreference(keyStrict))
+        radioStrict.onInfoClickListener {
+            nav(
+                R.id.trackingProtectionFragment,
+                TrackingProtectionFragmentDirections
+                    .actionTrackingProtectionFragmentToTrackingProtectionBlockingFragment(true)
+            )
+        }
+        radioStrict.onClickListener {
+            with(requireComponents) {
+                val policy = core.createTrackingProtectionPolicy()
+                useCases.settingsUseCases.updateTrackingProtection.invoke(policy)
+                useCases.sessionUseCases.reload.invoke()
+            }
+        }
+    }
+
+    private fun bindRecommended() {
```

Comment:
Since this function and `bindStrict` are very similar, I think the functionality should be extracted to a helper function that takes the preference key as a parameter then returns the radio.